### DISCUSSION TITLE
Is it possible to perform classification and regression simultaneously in a single model training

### DISCUSSION THREAD

**Discussion by ogencoglu** [catboost/catboost#2673] [Q&A]
For example by having two targets, one with binary labels and other target with numerical. Is it possible to create custom loss function or utilize the well-known loss functions as a weighted sum for this kind of formulation?

**Comment by ek-ak** [comment]
Hello!
The only thing I can suggest is to implement you own custom **Multiregression** loss. 
https://github.com/catboost/tutorials/blob/54c6f8fcd5e19f3eaa686d1f1be638706d9b610c/custom_loss/custom_loss_and_metric_tutorial.ipynb
